.github/workflows/pkgdown.yaml

```diff
@@ -28,21 +28,20 @@ jobs:
         id: install-r
 
       - uses: r-lib/actions/setup-pandoc@v1
-      
+
       - uses: r-lib/actions/setup-tinytex@v1
 
       - name: Install pak and query dependencies
         run: |
-          install.packages("pak", repos = "https://r-lib.github.io/p/pak/dev/")
-          saveRDS(pak::pkg_deps("local::.", dependencies = TRUE), ".github/r-depends.rds")
-        shell: Rscript {0}
+          sudo R -q -e 'install.packages("pak", lib = .Library, repos = "https://r-lib.github.io/p/pak/dev/")'
```

Comment:
Why do you need the sudo here?

Reply:
It was an idea suggested by @gaborcsardi, because without sudo I was told the folder was not writable.
The whole problem in this workflow is that I want to install pak from its daily repo but pak is cached with all other packages and therefore the cached version overwrites the old version.
In the end I might just change the cache name.

Reply:
Bumping the `-1-` to `-2-` cleared the cache already, I guess? But this is a general problem with this workflow: if the cache is restored then an older version of pak is  potentially restored as well.

We tried to install pak into `.Library`, so it is not cached. But that apparently needs `sudo`, and then the cache files are owned by root, which causes different problems.

Reply:
In that case, I'd suggest eliminating everything from this PR except for the Needs field in the DESCRIPTION and opening an issue in r-lib/actions so we fix this upstream.